### PR TITLE
[DE] Translations Player-Turn

### PR DESCRIPTION
**Pull request by DerSchlupf**
FINAL VERSION:
image
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
Heyho, new work from me! I will fix the problems it in the next days. Feel free to make suggestions. :)

**Comment by DerSchlupf** [comment]
Heyho, 
I've shrinked the town-portal-immage and the shield-immage slightly, in order to have this final layout. I've shorted the text on page 3 to move the word "kaufen" to the left side. But it doesnt happen? I believe instead of moving "kaufe" to the left side, they have made the monk bigger.
Other than that: Do you have any other layout- or translation-suggestions? 
If not, in my opionin this section is ready for merge.

File: translations/player_turns.tex/de.po [hunk 1]
```diff
@@ -229,6 +314,14 @@ msgid ""
...
+"Sie hat 3 Karten aus der vorherigen Runde auf der Hand und möchte 2 davon abwerfen, bevor sie bis zum \\mbox{Handkartenlimit \\svg{hand}} nachzieht.\n"
+"Ihr Handkartenlimit ist 5, da ihr Hauptheld auf Stufe 3 ist. Also zieht sie nach dem Abwerfen noch 4 Karten (siehe \\pagelink{Level}{Stufeneffekte}).}\\par\n"
+"\\textit{Anschließend nutzt sie zuerst den Baumarker, um die \\svgunit{golden} Behausung zu bauen, und danach den Bevölkerungsmarker, um die \\svgunit{golden} Einheit Kämpe zu \\pagelink{Units}{rekrutieren}.\n"
+"Sie darf dies nur tun, da sie die niederen Behausungen (\\svgunit{bronze} und \\svgunit{silver}) bereits errichtet hat und genügend \\pagelink{Resources}{Ressourcen} für den Bau der Behausung und die Rekrutierung der Kämpen besitzt.}\\par\n"
+"\\textit{Auf diese Weise für den Kampf vorbereitet gibt sie einen BP aus, um ihren Helden auf das angrenzende Feld zu bewegen, welches vom Haupthelden Sandro ihres Mitspielers Bob besetzt ist.\n"
+"Alicia und Bob haben jetzt noch Gelegenheit weitere Stadtaktionen durchzuführen. Alicia hat aber bereits gebaut und rekrutiert und dürfte nur noch Zauber kaufen.}\\par\n"
```
Comment (qwrtln) [inline translations/player_turns.tex/de.po:323]:
```suggestion
"\\textit{Vorbereitet auf den Kampf gibt sie einen BP aus, um ihren Helden auf das angrenzende Feld zu bewegen, das vom Haupthelden Sandro ihres Mitspielers Bob besetzt ist.\n"
"Alicia und Bob haben noch Gelegenheit, weitere Stadtaktionen durchzuführen. Alicia hat jedoch bereits gebaut und rekrutiert und darf nur noch Zauber kaufen.}\\par\n"
```

Comment (qwrtln) [reply translations/player_turns.tex/de.po:323]: Will get rid of "kaufen" orphan.

Comment (DerSchlupf) [reply translations/player_turns.tex/de.po]: > Will get rid of "kaufen" orphan.

Its done! Thanx for your suggestions! :)

**Review by qwrtln — APPROVED** [review]

**Pull request merged**